specs/mediawiki/v1/content.yaml

```diff
@@ -110,7 +105,7 @@ paths:
         uri: /{domain}/sys/parsoid/html/{title}/
 
 
-  /{module:page}/{title}/html/{revision}{/tid}:
+  /{module:page}/html/{title}/{revision}{/tid}:
```

Comment:
Idea: accept either tids or revisions for the revision parameter, and add a separate entry point `/{module:page}/html/{title}/{revision}/{tid}` to list html tids for a given MW revision.